assets/css/news-banner.css

```
@@ -0,0 +1,118 @@
+
+.news-banner {
+  --news-banner-max: min(92rem, calc(100% - 1.5rem));
+  --news-banner-pad-x: clamp(0.75rem, 1.5vw, 1.35rem);
+  --news-banner-bg: var(--openms-chrome-banner-bg);
+  --news-banner-border: var(--openms-chrome-banner-border);
+  --news-banner-label-color: var(--openms-chrome-banner-label);
+  --news-banner-label-bg: var(--openms-chrome-banner-label-bg);
+  --news-banner-label-border: var(--openms-chrome-banner-label-border);
+  --news-banner-text-color: var(--openms-chrome-banner-text);
+  --news-banner-focus-ring: rgba(var(--openms-blue-secondary-rgb), 0.45);
+  position: relative;
```

Stylelint (17.13.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)